docs/api/covidcast-signals/changehc.md

```diff
@@ -0,0 +1,159 @@
+---
+
+# Doctor Visits
+{: .no_toc}
+
+* **Source name:** `changehc`
+* **Number of data revisions since 19 May 2020:** 0
+* **Date of last change:** Never
+* **Available for:** county, hrr, msa, state (see [geography coding docs](../covidcast_geography.md))
+
+This data source is based on information about outpatient visits, provided to us
+by health system partners. Using this outpatient data, we estimate the
+percentage of COVID-related doctor's visits in a given location, on a given day.
+
+| Signal | Description |
+| --- | --- |
+| `smoothed_cli` | Estimated percentage of outpatient doctor visits with confirmed COVID-19, based on data from Change Healthcare, smoothed in time using a Gaussian linear smoother |
+| `smoothed_adj_cli` | Same, but with systematic day-of-week effects removed; see [details below](#day-of-week-adjustment) |
+
+## Table of contents
+{: .no_toc .text-delta}
+
+1. TOC
+{:toc}
+
+## Lag and Backfill
+
+Note that because doctor's visits may be reported to Change Healthcare
+several days after they occur, these signals are typically available with
+several days of lag. This means that estimates for a specific day are only
+available several days later.
+
+The amount of lag in reporting can vary, and not all visits are reported with
+the same lag. After we first report estimates for a specific date, further data
+may arrive about outpatients visits on that date. When this occurs, we issue new
+estimates for those dates. This means that a reported estimate for, say, June
+10th may first be available in the API on June 14th and subsequently revised on
+June 16th.
+
+## Limitations
+
+This data source is based on outpatient visit data provided to us by Change
+Healthcare. Change Healthcare reports on a portion of United States
+outpatient doctor's visits, but not all of them, and so this source only
+represents those visits known to them. Their coverage may vary across the United
+States, but they report on about 45% of all doctor's visits nationally.
+
+Standard errors are not available for this data source.
+
+Due to changes in medical-seeking behavior on holidays, this data source has
+upward spikes in the fraction of doctor's visits that are COVID-related around
+major holidays (e.g. Memorial Day, July 4, Labor Day, etc.). These spikes are
+not necessarily indicative of a true increase of COVID-like illness in a
+location.
+
+## Qualifying Conditions
+
+We receive data on the following two categories of counts:
+
+- Denominator: Daily count of all unique outpatient visits.
+- Covid: Daily count of all unique visits with primary ICD-10 code in any of:
+{U071, U072, B9721, or B9729}.
+
+## Estimation
+
+### COVID-Like Illness
+
+For a fixed location $i$ and time $t$, let $Y_{it}$
+denote the Covid counts and let $N_{it}$ be the
+total count of visits (the *Denominator*). Our estimate of the CLI percentage is
+given by
+
+$$
+\hat p_{it} = 100 \cdot  \frac{Y_{it}}{N_{it}}
+$$
+
+The estimated standard error is:
```

Comment:
Is it weird to include this definition if we do not make SE available for this signal in the API?

Reply:
I wasn't sure. The definition is in the DV docs and SE isn't included there either.

Reply:
Probably was written up while we were deciding if we can or can't include it. May as well remove it from both

Reply:
Removed